web/src/js/components/Header/ConnectionIndicator.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from "react"
```

Comment:
Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.

I will change its in a different pull request...